schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -671,6 +671,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_eventhubs_authorizationRules" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_eventhubs_consumergroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_disasterRecoveryConfigs" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-01-01-preview/Microsoft.EventHub.json#/resourceDefinitions/namespaces" },
```

Comment:
>namespaces [](start = 137, length = 10)

we have a new root schema: 2019-04-01/deploymentTemplate.json. Can you add them to that root schema too?

Reply:
Resolved